projects/plugins/jetpack/extensions/blocks/videopress/edit.js

```diff
@@ -451,43 +432,70 @@ const VideoPressEdit = CoreVideoEdit =>
 			return (
 				<Fragment>
 					{ blockSettings }
-					<BlockFigureWrapper
-						className={ classnames( 'wp-block-video', className, videoPressClassNames, {
-							[ `align${ align }` ]: align,
-						} ) }
-					>
-						<div className="wp-block-embed__wrapper">
-							<SandBox html={ html } scripts={ scripts } type={ videoPressClassNames } />
-						</div>
-
-						{
-							/*
-							Disable the video player when the block isn't selected,
-							so the user clicking on it won't play the
-							video when the controls are enabled.
-						*/
-							! interactive && (
-								<div
-									className="block-library-embed__interactive-overlay"
-									onMouseUp={ this.hideOverlay }
-								/>
-							)
-						}
-						{ ( ! RichText.isEmpty( caption ) || isSelected ) && (
-							<RichText
-								tagName="figcaption"
-								placeholder={ __( 'Write caption…', 'jetpack' ) }
-								value={ caption }
-								onChange={ value => setAttributes( { caption: value } ) }
-								inlineToolbar
-							/>
-						) }
-					</BlockFigureWrapper>
+					<VpBlock
+						{ ...this.props }
+						hideOverlay={ this.hideOverlay }
+						html={ html }
+						scripts={ scripts }
+						interactive={ interactive }
+						caption={ caption }
+					/>
 				</Fragment>
 			);
 		}
 	};
 
+// The actual, final rendered video player markup
+// In a separate function component so that `useBlockProps` could be called.
+const VpBlock = props => {
```

Comment:
Totally not a blocker, but what if this was `VideoBlock` or `VideoPressBlock`, it just looks a little funny to my eyes.